app/uk/gov/hmrc/tai/service/JrsService.scala

```diff
@@ -32,17 +32,13 @@ class JrsService @Inject()(jrsConnector: JrsConnector, appConfig: ApplicationCon
 
   def getJrsClaims(nino: Nino)(implicit hc: HeaderCarrier): OptionT[Future, JrsClaims] = OptionT {
     jrsConnector
-      .getJrsClaims(nino)(hc)
-      .fold[Option[JrsClaims]](None)(
-        jrsClaimsData => extractClaimsData(jrsClaimsData)
-      )
+      .getJrsClaimsForIndividual(nino)(hc)
+      .map(JrsClaims(appConfig, _))
+      .getOrElse(None)
   }
 
-  private def extractClaimsData(jrsClaimsData: JrsClaims) =
-    if (jrsClaimsData.employers.nonEmpty) Some(JrsClaims(appConfig, jrsClaimsData.employers)) else None
-
   def checkIfJrsClaimsDataExist(nino: Nino)(implicit hc: HeaderCarrier): Future[Boolean] =
     if (appConfig.jrsClaimsEnabled) {
-      jrsConnector.getJrsClaims(nino)(hc).fold(false)(jrsClaims => true)
+      jrsConnector.getJrsClaimsForIndividual(nino)(hc).map(_ => true).getOrElse(false)
```

Comment:
Is this just `.contains(true)`?

Reply:
Its to check if we are getting Some(jrsClaimsData) from connector then return true else return false